icu4c/source/test/intltest/locnmtst.cpp

```diff
@@ -420,6 +421,23 @@ void LocaleDisplayNamesTest::TestRootEtc() {
   delete ldn;
 }
 
+void LocaleDisplayNamesTest::TestNumericRegionID() {
+    UErrorCode err = U_ZERO_ERROR;
+    ULocaleDisplayNames* ldn = uldn_open("es_MX", ULDN_STANDARD_NAMES, &err);
+    UChar displayName[200];
+    int32_t displayNameLength = uldn_regionDisplayName(ldn, "019", displayName, 200, &err);
+    test_assert(U_SUCCESS(err));
+    test_assert_equal(UnicodeString(u"América"), UnicodeString(displayName));
+    uldn_close(ldn);    
+
+    err = U_ZERO_ERROR; // reset in case the test above returned an error code
+    ldn = uldn_open("en_AU", ULDN_STANDARD_NAMES, &err);
+    displayNameLength = uldn_regionDisplayName(ldn, "002", displayName, 200, &err);
```

Comment:
ditto

Reply:
Yeah, you're probably right.  Is it important enough to sit through another hour-plus of pre-commit checks, though?

Reply:
Up to you, that's why I approved.
On one hand, it's not really important.
On the other hand, it's fine if it gets submitted tomorrow.

Reply:
Okay.  In the interest of time, I'm going to leave it.  I'll try to remember to clean that up when I go back in there to do the "real" fix, which hopefully won't be that long from now.